src/components/EditableLabel/EditableLabel.tsx

```diff
@@ -1,23 +1,31 @@
-import { CSSProperties, FormEvent, KeyboardEvent, useRef, useState } from "react";
+import { useEffect, CSSProperties, FormEvent, KeyboardEvent, useRef, useState } from "react";
 import styled from "styled-components";
 
 export interface RootBodyProps {
   value?: string;
+  placeholder?: string;
   onBlur?: (value: string) => void;
+  onChange?: (inputStr: string) => void;
   className?: string;
   style?: CSSProperties;
 }
 
 export default function EditableLabel({
   value: initValue = "",
   onBlur = () => {},
+  onChange,
```

Comment:
Looks good to add onChange, do you want to follow same pattern with onBlur?
```
onChange = () => {}
```

Reply:
Sure thing

Reply:
https://github.com/ryonryon/trello-clone/pull/67/commits/d65dad3a1d1a4656d877768989c9b8dff0633cc9